app.js

```diff
@@ -104,6 +109,7 @@ app.post("/api/shorten",
         const data = {
             url,
             user: req.user.user,
+            expires: expires ? expires : null,
```

Comment:
could do `expires ?? null`

Reply:
Fixed in [020bc87](https://github.com/datasektionen/pico/pull/15/commits/020bc87270b52ccb09436ed6a52bddbcaba0ad7e)